code/modules/mob/living/living.dm

```diff
@@ -787,17 +793,9 @@ Thanks.
 				hook.override_starting_X--
 				hook.override_target_X--
 
-/mob/living
-    var/event/on_resist
-
 /mob/living/New()
```

Comment:
Because oldcode
And more specifically, because I forgot to remove this line when moving code over to one definition.